content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/_cq_design_dialog/.content.xml

```diff
@@ -51,9 +51,6 @@
                                         text="Enable DM features"
                                         uncheckedValue="false"
                                         value="{Boolean}true">
-                                        <granite:rendercondition jcr:primaryType="nt:unstructured"
```

Comment:
I'm not sure if this is the way to go. This means that DM features would be always available even if no scene7 or NGDM is available. Is that what you want?

Reply:
@LSantha Is there a rendercondition to check for OSGI configuration's value in core components? Or any other way apart from renderconditions?

FWIW, removal of this rendercondition will only make the enable DM features checkbox visible in the design dialog of image component. It will not enable it by default. An author will have to update the template to enable the feature manually.

Reply:
@mohiaror , I don't think there is such render condition but you can create one.

I know this thing is about the design dialog but as you can see it was still provided with a render condition to avoid any confusion for customer not using this feature.

Reply:
@LSantha I have added an OR condition for both DM and NGDM. Please see if it looks Ok to you.